pkgs/development/python-modules/aioitertools/default.nix

```
@@ -0,0 +1,33 @@
+{ buildPythonPackage
+, fetchPypi
+, pythonOlder
+, pythonAtLeast
+, typing-extensions
+, lib
+}:
+
+buildPythonPackage rec {
+  pname = "aioitertools";
+  version = "0.7.0";
```

```suggestion
  version = "0.7.1";
```
aioitertools has since been updated on [pypi](https://pypi.org/project/aioitertools/)